client/scss/components/_workflow-tasks.scss

```diff
@@ -1,30 +1,30 @@
 .workflow-tasks {
-  $task-width: 117px;
-  $task-height: 56px;
+  $task-width: 7.313rem;
+  $task-height: 3.5rem;
 
   list-style-type: none;
 
   &__task {
     display: inline-block;
     background-color: theme('colors.secondary.50');
-    border: 2px solid theme('colors.secondary.100');
-    border-radius: 5px;
+    border: 0.125rem solid theme('colors.secondary.100');
+    border-radius: 0.313rem;
```

Comment:
I don’t see why we would want the border width or radius to change depending on the font size. Could you explain why you did this?